v22.2/sso-db-console.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Single Sign-on for DB Console
+title: Single Sign-on (SSO) for DB Console
```

Comment:
The name change of this page: do we want to ensure that users can use the dropdown to switch to older versions despite the name change? Right now, on the preview, it seems like a user won't be able to do that. I had this same issue and resolved it with info in this thread: https://cockroachlabs.slack.com/archives/C01PHNMUFLN/p1649943047280749

Reply:
ok, I think I fixed this by adding `key: sso.html`, let's see if that does it.